ipatests/test_integration/test_adtrust_install.py

```diff
@@ -199,3 +199,14 @@ def test_add_agent_on_running_replica_with_compat(self):
         entry = conn.get_entry(DN(
             "cn=groups,cn=Schema Compatibility,cn=plugins,cn=config"))
         assert entry.single_value['schema-compat-lookup-nsswitch'] == "group"
+
+        """Test if schema-compat-entry-attribute is set
+
+        This is to ensure if said entry is set after installation.
+
+        related: https://pagure.io/freeipa/issue/8193
+        """
+        entry_list = list(entry['schema-compat-entry-attribute'])
```

Comment:
@mrizwan93 
Check for compat tree enablement is missing.
Also this only perform check for installation with AD . We need to check without AD install as well.

Reply:
The check is already covered at https://github.com/freeipa/freeipa/pull/4324/files#diff-295adcfce4b5bafad3de8b3a88ca533dR201

I'll submit another commit for non AD environment.